packages/vsce/package.json

```diff
@@ -866,9 +866,20 @@
     "tsupDep": "node ./scripts/tsup-deps.js",
     "getPrebuilds": "node ./scripts/getSecretsPrebuilds.js",
     "package": "vsce package --dependencies --allow-star-activation && node ../../scripts/mv-pack.js cics-extension-for-zowe vsix",
+    "test:e2e:prepare": "extest get-vscode -s .cics && extest get-chromedriver -s .cics",
+    "test:e2e:install": "extest install-from-marketplace zowe.vscode-extension-for-zowe -s .cics -e .cics",
+    "test:e2e:infra:setup": "mkdir -p ~/e2e-global && export ZOWE_CLI_HOME=~/e2e-global",
```

Comment:
I'm still of the opinion we should be creating the test ZOWE_CLI_HOME directory in the same place as our test workspace folder (`.cics`) - we could have `.cics-project` and `.cics-global` folders, both under __e2e__. I think it would be bad practice to create folders under a user's home directory.

Happy to be disuaded here, what do people think 🤷‍♂️

Reply:
I like the addition of this though - using a separate ZOWE_CLI_HOME seems a safe option

Reply:
Yes - keeping everything contained within the test directory means there's no chance of different test runs conflicting, one leaving bad data and causing a subsequent test to fail in a confusing way.

However I can't find any other references to `ZOWE_CLI_HOME` or `e2e-global` so I'm not sure if they're being used yet or if they're just "ready to go".